docs/src/main/asciidoc/dynamodb-guide.adoc

```diff
@@ -0,0 +1,751 @@
+Keep in mind it's actively developed and does not support yet all the features available in SDK 1.x such as https://github.com/aws/aws-sdk-java-v2/issues/36[Document APIs], https://github.com/aws/aws-sdk-java-v2/issues/35[Object Mappers]
+
+The Quarkus extension supports two programming models:
+
+* Blocking access using Apache Http Client
+* https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/basics-async.html[Asynchronous programming] based on JDK's `CompletableFuture` objects and Netty Http client.
+
+In this guide, we see how you can get your REST services to use the DynamoDB locally and on AWS account.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* an IDE
+* Apache Maven 3.5.3+
+* AWS Account to access DynamoDB service
+* Optionally, Docker for your system to run dynamoDB locally for testing purposes
+
+=== Setup DynamoDB locally
+
+The easiest way to start work with DynamoDB is to run local instance as container.
+
+[source,shell,subs="verbatim,attributes"]
+----
+docker run --publish 8000:8000 amazon/dynamodb-local:1.11.477 -jar DynamoDBLocal.jar -inMemory -sharedDb
+----
+
+This starts a DynamoDB instance, that's accessible on port `8000`.
+You can check it running accessing the web shell on `http://localhost:8000/shell`.
+
+Have a look at the https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html[Setting Up DynamoDB Local] for other options to run DynamoDB.
+
+Open `http://localhost:8000/shell` in your browser.
+
+Copy & paste a following code to the shell and run it.
+[source,js,subs="verbatim,attributes"]
+----
+var params = {
+    TableName: 'QuarkusFruits',
+    KeySchema: [{ AttributeName: 'fruitName', KeyType: 'HASH' }],
+    AttributeDefinitions: [{  AttributeName: 'fruitName', AttributeType: 'S', }],
+    ProvisionedThroughput: { ReadCapacityUnits: 1, WriteCapacityUnits: 1, }
+};
+
+dynamodb.createTable(params, function(err, data) {
+    if (err) ppJson(err);
+    else ppJson(data);
+
+});
+----
+
+=== Setup Dynamodb on AWS
+
+Before you can use the AWS SDKs with DynamoDB, you must get an AWS access key ID and secret access key.
+For more information, see [Setting Up DynamoDB (Web Service)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SettingUp.DynamoWebService.html).
+
+We recommend to use AWS CLI to provision table
+
+[source,shell,subs="verbatim,attributes"]
+----
+aws dynamodb create-table --table-name QuarkusFruits \
+                          --attribute-definitions AttributeName=fruitName,AttributeType=S \
+                          --key-schema AttributeName=fruitName,KeyType=HASH \
+                          --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1
+----
+
+== Solution
+The application built is similar as in other guides and allows to manage elements (fruits) and they are stored in DynamoDB.
```

Comment:
```suggestion
The application built here allows to manage elements (fruits) stored in Amazon DynamoDB.
```
The user might not have read the other guides because he might just be interested in DynamoDB.